# 11 Inventory surprise proxy

Is the holdout correlation between this mean-baseline stock surprise and the nominal EIA Wednesday RWTC return negative?

Holdout starts 2020-01-01. Verdict: failed.

Weekly change in EIA WCESTUS1, kept only when the week-ending dates are 7 days apart, minus the mean of the prior 52 kept changes. The mean uses weeks strictly before the current week. This is a trailing baseline, not an analyst survey.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'inv-surprise')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
Train correlation of the mean-baseline surprise with the release-day RWTC log return is -0.098. Holdout correlation is 0.022. The holdout correlation is not negative. The bar was missed.
WCESTUS1 change minus the prior 52-week mean, versus the Wednesday RWTC log return
Train | 1584 | -0.098
Holdout | 348 | 0.022
Skipped, no Wednesday RWTC print | 307 | 


## Method

The EIA week ends Friday. The nominal release is the Wednesday five days later. If that Wednesday has no RWTC print, the week is skipped. A Thursday holiday release is not moved.

## Result

Train correlation of the mean-baseline surprise with the release-day RWTC log return is -0.098. Holdout correlation is 0.022. The holdout correlation is not negative. The bar was missed.